src/som/interpreter/Bytecodes.java

```diff
@@ -78,29 +80,35 @@ public static String getPaddedBytecodeName(byte bytecode) {
     return PADDED_BYTECODE_NAMES[bytecode];
   }
 
-  public static int getBytecodeLength(byte bytecode) {
+  public static final int getBytecodeLength(final byte bytecode) {
```

Comment:
Think `final` might not be necessary/redundant. And checkstyle would complain about it.

Reply:
I explicitly "checkstyled" the whole project before committing and did not get any complaints. I'll check whether I am using the right files

Reply:
actually, why should it complain? I checked my configuration and it seems similar to SOMns except the modified bits about tests/copyrights I had to remove to make intellij-checkstyle happy

Reply:
My assumption is that static methods cannot be overridden, which makes the final modifier redundant, I think.

Reply:
Ha yes indeed